email-common/src/main/java/io/commercetools/sunrise/email/MessageEditor.java

```diff
@@ -0,0 +1,21 @@
+package io.commercetools.sunrise.email;
+
+import javax.annotation.Nonnull;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * A functional interface for modifying a message without needing to handle checked exceptions like the
+ * {@link MessagingException}s thrown by {@link MimeMessage}.
+ */
+@FunctionalInterface
+public interface MessageEditor {
```

Comment:
I would guess that creating a POJO that can be converted to MimeMessage takes a considerable amount of time given how many methods the MimeMessage class of the Java Mail API and the related classes in Apache Commons Email have.

Given the drawbacks of Apache Commons Email mixing multiple responsibilities in the Email class, would you be fine with improving the JavaDoc of MessageEditor, as Laura suggested?